composer/callbacks/checkpoint_saver.py

```diff
@@ -303,11 +304,16 @@ def epoch_checkpoint(self, state: State, logger: Logger):
                 logger,
             )
 
-    def get_state_dict(self, state):
```

Comment:
This looked orphaned, long since moved to `composer/utils/checkpoint.py`

Reply:
Yeah, no idea what this is from. Don't think its used.